test/ts/Component-test.tsx

```diff
@@ -1,11 +1,11 @@
-import "mocha";
```

Comment:
I think Prettier got a hold of all the `.ts` files in the `ts` directory . I didn't edit these. My focus is manually on our other tests